curated_transformers/tests/generation/test_dolly_v2.py

```diff
@@ -27,7 +17,8 @@ def test_generate_deterministic():
         "What is the Rust programming language?",
         "What is spaCy?",
     ]
-    assert _collect_pieces(generator(prompts), len(prompts)) == [
+    print(generator(prompts))
```

Comment:
`stdout` is captured by default, so this shouldn't affect `pytest` output, but I wanted to flag this just in case it's a debug statement.

Reply:
Thanks! Yeah, these were debug statements, removed.